### ISSUE TITLE
Found a data link escape.

### ISSUE THREAD

**Issue by JulienPalard** [python/python-docs-pl#15]
The PDF build tells:
```
! Emergency stop.
l.1704 Na przykład załóżmy, ^^P
                                   że chcemy stworzyć listę kwadratów, j...
```

`^P` according to an ASCII table conveninetly presented in two columns is `DLE`, or `\x10`:

    020   16    10    DLE (data link escape)      120   80    50    P 

and `git grep` find it:

    $ git grep $'\x10'
    tutorial/datastructures.po:msgstr "Na przykład załóżmy, ^Pże chcemy stworzyć listę kwadratów, jak tu::"

I bet this is a kind of typo?

**Comment by m-aciek** [comment]
Yes, apparently the character is not printable on MacOS and printable on Windows, I don't know how it showed up.

I am sorry for the late reply. I was able to [reproduce the error](https://github.com/python/python-docs-pl/runs/1659001147?check_suite_focus=true#step:4:115684) in [CI](https://github.com/python/python-docs-pl/commit/52d82cb2ef6185b87cdaaecfa196fe66ce5e407b) (from now on errors like this should be discovered in the repository).

The issue [is solved for 3.9](https://github.com/python/python-docs-pl/commit/312924538fe9be71481443fca705fb4aecd35d15) and the change should be soon also fetched and committed for 3.8 branch.

Thank you for the heads up!

**Comment by m-aciek** [comment]
PDFs generate now successfully for both 3.8 and 3.9.
